integration-tests/utils/aws/dynamoDB.ts

```diff
@@ -0,0 +1,187 @@
+import {
+  DeleteItemCommand,
+  GetItemCommand,
+  PutItemCommand
+} from '@aws-sdk/client-dynamodb'
+
+import { AUDIT_REQUEST_DYNAMODB } from '../../constants/awsParameters'
+import {
+  ZendeskFormFieldIDs,
+  ZENDESK_END_USER_EMAIL,
+  ZENDESK_END_USER_NAME
+} from '../../constants/zendeskParameters'
+import { dynamoDBTicketDetails } from '../../constants/dynamoDBTicketDetails'
+import { dynamoDBClient } from './dynamoDBClient'
+
+export const populateDynamoDBWithTestItemDetails = async (ticketID: string) => {
+  const ticketDetails = dynamoDBTicketDetails.ticket
+
+  const populateTableParams = {
+    TableName: AUDIT_REQUEST_DYNAMODB,
+    ReturnValues: 'ALL_OLD',
+    Item: {
+      zendeskId: { S: `${ticketID}` },
+      requestInfo: {
+        M: {
+          zendeskId: { S: `${ticketID}` },
+          dateFrom: {
+            S: `${getFieldValue(
+              ticketDetails,
+              ZendeskFormFieldIDs.PII_FORM_REQUEST_DATE_FIELD_ID
+            )}`
+          },
+          dateTo: {
+            S: `${getFieldValue(
+              ticketDetails,
+              ZendeskFormFieldIDs.PII_FORM_REQUEST_DATE_FIELD_ID
+            )}`
+          },
+          identifierType: {
+            S: `${getFieldValue(
+              ticketDetails,
+              ZendeskFormFieldIDs.PII_FORM_IDENTIFIER_FIELD_ID
+            )}`
+          },
+          recipientEmail: {
+            S: `${getFieldValue(
+              ticketDetails,
+              ZendeskFormFieldIDs.PII_FORM_IDENTIFIER_RECIPIENT_EMAIL
+            )}`
+          },
+          recipientName: {
+            S: `${getFieldValue(
```

Comment:
I don't think we need these  `getFieldValue` and `getFieldListValues` functions if we're just referencing a static object?